packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -903,7 +903,13 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
           newIndex = item.index;
         }
       } else {
-        if (itemStart <= proxyItemStart && proxyItemStart <= itemMiddle) {
+        if (item.index == _dragIndex!) {
```

Comment:
Would we not reach here based on the `continue` above?

Reply:
No, because that will only `continue` if `_reverse` is true.